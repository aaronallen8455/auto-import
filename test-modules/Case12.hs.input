module Case12 where

main :: IO ()
main = f1 b2 Bar baz
