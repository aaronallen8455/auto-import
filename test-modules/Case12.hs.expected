module Case12 where

import Foo (Bar, Foo(f1, b2), baz)

main :: IO ()
main = f1 b2 Bar baz
